components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -228,4 +229,63 @@ export function prebuildStatusIcon(status: PrebuiltWorkspaceState | undefined) {
         default:
             break;
     }
+}
+
+export function PrebuildInstanceStatus(props: { prebuildInstance?: WorkspaceInstance, isDark?: boolean }) {
+    let status = <></>;
+    let details = <></>;
+    switch (props.prebuildInstance?.status.phase) {
+        case undefined: // Fall through
+        case 'unknown':
+            status = <div className="flex space-x-1 items-center text-yellow-600">
+                <img className="h-4 w-4" src={StatusPaused} />
+                <span>PENDING</span>
+                </div>;
+            details = <div className="flex space-x-1 items-center text-gray-400">
+                <img className="h-4 w-4 animate-spin" src={props.isDark ? SpinnerDark : Spinner} />
+                <span>Prebuild in progress ...</span>
+                </div>;
+            break;
+        case 'preparing': // Fall through
+        case 'pending': // Fall through
+        case 'creating': // Fall through
+        case 'initializing': // Fall  through
+        case 'running': // Fall through
+        case 'interrupted':
+            status = <div className="flex space-x-1 items-center text-blue-600">
+                <img className="h-4 w-4" src={StatusRunning} />
+                <span>RUNNING</span>
+                </div>;
+            details = <div className="flex space-x-1 items-center text-gray-400">
+                <img className="h-4 w-4 animate-spin" src={props.isDark ? SpinnerDark : Spinner} />
+                <span>Prebuild in progress ...</span>
+                </div>;
+            break;
+        case 'stopping': // Fall through
+        case 'stopped':
+            status = <div className="flex space-x-1 items-center text-green-600">
+                <img className="h-4 w-4" src={StatusDone} />
+                <span>READY</span>
+                </div>;
+            details = <div className="flex space-x-1 items-center text-gray-400">
+                <img className="h-4 w-4 filter-grayscale" src={StatusRunning} />
+                <span>{!!props.prebuildInstance?.stoppedTime
+                    ? `${Math.round(((new Date(props.prebuildInstance.stoppedTime).getTime()) - (new Date(props.prebuildInstance.creationTime).getTime())) / 1000)}s`
+                    : '...'}</span>
+                </div>;
+            break;
+    }
+    if (props.prebuildInstance?.status.conditions.failed) {
+        status = <div className="flex space-x-1 items-center text-gitpod-red">
+            <img className="h-4 w-4" src={StatusFailed} />
+            <span>FAILED</span>
+            </div>;
+        details = <div className="flex space-x-1 items-center text-gray-400">
+            <span>Prebuild failed</span>
+            </div>;
+    }
+    return <div className="flex flex-col space-y-1 justify-center text-sm font-semibold">
+        <div>{status}</div>
```

Comment:
**question**: Now that we have proper status updates down here and this component is used as is in the project configurator, wondering if it makes sense to change or remove the status from the header although it could be useful to have the status up there.

If we remove the header instance we'll have to make sure the bottom part of the prebuild output is always visible. Of course, we could leave it as is for now but make sure the designs for both is identical. Alternatively, we could only include the status icon to avoid having duplicated information. What do you think? 

| BEFORE | AFTER (A) | AFTER (B) |
|-|-|-|
| <img width="553" alt="BEFORE" src="https://user-images.githubusercontent.com/120486/131981786-f5f58550-55dc-4f87-b55a-cb365dc13e53.png"> | <img width="553" alt="AFTER (A)" src="https://user-images.githubusercontent.com/120486/131982306-9bc20c00-ab65-455a-8b7a-ab6180146619.png"> | <img width="553" alt="AFTER (B)" src="https://user-images.githubusercontent.com/120486/131981792-12c76bd7-0d67-45bb-9280-3e3a1c57e790.png"> |

Reply:
Excellent point and proposals! Filed as https://github.com/gitpod-io/gitpod/issues/5526